src/frontend/ForumIndex.css

```diff
@@ -0,0 +1,30 @@
+.headerContainer {
```

Comment:
Good positioning, didn't need to add any colors as we can do that later, but the layout is correct 👍